### ISSUE TITLE
Please support non-standard App/User primary key

### ISSUE THREAD

**Issue by chops888** [haruncpi/laravel-user-activity#33]
please offer a config variable for an alternate primaryKey for App/User

instead of hard-coded ->id

**Comment by FarhanShares** [comment]
Yeah, if used with UUID/ULID in the user model, it just fails. A config for this or migration publishing would be nice.
